airflow/www/static/js/dag/details/dagRun/index.tsx

```diff
@@ -74,85 +74,100 @@ const DagRun = ({ runId }: Props) => {
   const detailsLink = appendSearchParams(dagRunDetailsUrl, detailsParams);
 
   return (
-    <Box py="4px">
-      <Flex justifyContent="space-between" alignItems="center">
-        <Button as={Link} variant="ghost" colorScheme="blue" href={detailsLink}>DAG Run Details</Button>
-        <Button as={Link} variant="ghost" colorScheme="blue" href={graphLink} leftIcon={<MdOutlineAccountTree />}>
-          Graph
-        </Button>
-        <MarkFailedRun dagId={dagId} runId={runId} />
-        <MarkSuccessRun dagId={dagId} runId={runId} />
-      </Flex>
-      <Divider my={3} />
-      <Flex justifyContent="flex-end" alignItems="center">
-        <Text fontWeight="bold" mr={2}>Re-run:</Text>
-        <ClearRun dagId={dagId} runId={runId} />
-        <QueueRun dagId={dagId} runId={runId} />
-      </Flex>
-      <Divider my={3} />
-      <Flex alignItems="center">
-        <Text as="strong">Status:</Text>
-        <SimpleStatus state={state} mx={2} />
-        {state || 'no status'}
-      </Flex>
-      <br />
-      <Text whiteSpace="nowrap">
-        Run Id:
-        {' '}
-        <ClipboardText value={runId} />
-      </Text>
-      <Text>
-        Run Type:
-        {' '}
-        {runType === 'manual' && <MdPlayArrow style={{ display: 'inline' }} />}
-        {runType === 'backfill' && <RiArrowGoBackFill style={{ display: 'inline' }} />}
-        {runType === 'scheduled' && <MdOutlineSchedule style={{ display: 'inline' }} />}
-        {runType}
-      </Text>
-      <Text>
-        Duration:
-        {' '}
-        {formatDuration(getDuration(startDate, endDate))}
-      </Text>
-      {lastSchedulingDecision && (
-      <Text>
-        Last Scheduling Decision:
-        {' '}
-        <Time dateTime={lastSchedulingDecision} />
-      </Text>
-      )}
-      <br />
-      {startDate && (
-      <Text>
-        Started:
-        {' '}
-        <Time dateTime={startDate} />
-      </Text>
-      )}
-      {endDate && (
-      <Text>
-        Ended:
-        {' '}
-        <Time dateTime={endDate} />
-      </Text>
-      )}
-      {dataIntervalStart && dataIntervalEnd && (
-        <>
-          <br />
-          <Text as="strong">Data Interval:</Text>
-          <Text>
-            Start:
-            {' '}
-            <Time dateTime={dataIntervalStart} />
-          </Text>
-          <Text>
-            End:
-            {' '}
-            <Time dateTime={dataIntervalEnd} />
-          </Text>
-        </>
-      )}
-    </Box>
+    <>
+      <Box>
```

Comment:
I don't think we need this `<Box>` as the Flex wraps all the children already

Reply:
you mean the one that opens on 78 and closes on 87?